src/components/TodoInfo/TodoInfo.jsx

```diff
@@ -1,4 +1,29 @@
-// Add the required props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import './TodoInfo.scss';
+import { UserInfo } from '../UserInfo';
+
+const cn = require('classnames');
+
+export const TodoInfo = ({ todo }) => {
+  const {
+    title,
+    completed,
+    user,
+  } = todo;
+
+  return (
+    <article
+      className={
+        cn({
+          TodoInfo: true,
+          'TodoInfo--completed': completed,
+        })
```

Comment:
you don't need to compare if completed is true, because it already gives the result of true or false.